src/equations/svaerd_kalisch_1d.jl

```diff
@@ -327,7 +326,7 @@ number of nodes as length of the second dimension.
     eta = view(q, 1, :)
     v = view(q, 2, :)
     D = view(q, 3, :)
-    beta_hat = equations.beta * (eta .+ D) .^ 3
```

Comment:
This was actually a bug. It should have been `D` already before. That's also why there are some changes in the change of the modified entropy in the test values.

Reply:
👍